src/gpio_messaging.toit

```diff
@@ -0,0 +1,38 @@
+import serialization
```

Comment:
You should add a copyright notice to the file.

Reply:
Thanks, will do that